src/Pages/Main/Components/SearchMenu.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+import SearchBar from '../../../Components/Search/SearchBar';
+
+function SearchMenu() {
+  const [isRoundTrip, setIsRoundTrip] = useState(true);
+
+  const tabBtnProps = {
+    roundTrip: {
+      clicked: isRoundTrip,
+      onClick: () => {
+        setIsRoundTrip(true);
+      },
+    },
+    onwayTrip: {
+      clicked: !isRoundTrip,
+      onClick: () => {
+        setIsRoundTrip(false);
+      },
+    },
+  };
+
+  return (
+    <Container>
+      <Tab>
+        <TabBtn {...tabBtnProps.roundTrip}>왕복</TabBtn>
+        <TabBtn {...tabBtnProps.onwayTrip}>편도</TabBtn>
+      </Tab>
```

Comment:
```suggestion
function SearchMenu() {
  const [isRoundTrip, setIsRoundTrip] = useState(true);

  const setRoundTrip = () => setIsRoundTrip(true)
  const setOneWayTrip = () => setIsRoundTrip(false)

  return (
    <Container>
      <Tab>
        <TabBtn clicked={isRoundTrip} onClick={setRoundTrip}>왕복</TabBtn>
        <TabBtn clicked={!isRoundTrip} onClick={setOneWayTrip}>편도</TabBtn>
      </Tab>
```
지금 보니 이런 방식이 더 간단하겠네요...